reactome-jenkins-release-config-template.properties

```diff
@@ -0,0 +1,54 @@
+releaseNumber=
+
+pathToSpeciesConfig=
+pathToOrthopairs=
+pathToWebsiteFilesUpdateFolder=
+pathToSkipList=
+dateOfRelease=yyyy-mm-dd
+
+pathToSpeciesConfig=
+pantherFileURL=ftp://ftp.pantherdb.org/ortholog/current_release/
+pantherQfOFilename=QfO_Genome_Orthologs.tar.gz
+pantherHCOPFilename=Orthologs_HCOP.tar.gz
+MGIFileURL=http://www.informatics.jax.org/downloads/reports/HGNC_homologene.rpt
+RGDFileURL=ftp://ftp.rgd.mcw.edu/pub/data_release/GENES_RAT.txt
+XenbaseFileURL=ftp://ftp.xenbase.org/pub/GenePageReports/GenePageEnsemblModelMapping.txt
+ZFINFileURL=https://zfin.org/downloads/ensembl_1_to_1.txt
+
+brenda.password=
+orphanet.user=
+cosmic.user=
+brenda.user=
+orphanet.password=
+cosmic.password=
+dockblaster.password=
+dockblaster.user=
+
+absoluteReleaseDirectoryPath=
+releaseDownloadDirectoryPath=
+protegeexporter.pathToWrapperScript=
+# 5 is for an 8-core system (need to leave some resources for MySQL and the OS).
+# If you have a different number of cores, you can change this value, but try to 
+# leave at least 2 or 3 cors available for other processes (such as MySQL and OS). 
```

Comment:
Type - cores?

Reply:
Haha, don't call me out on my typos in talking about typos :).

Reply:
:P